docs/core/execution-context.md

```
@@ -0,0 +1,354 @@
+  code: 'VALIDATION_FAILED',
+  message: 'Invalid input',
+  correlationId: ctx.correlationId,
+  meta: { tenantId: ctx.tenantId },
+});
+```
+
+---
+
+## File Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)